examples/llm_and_nlp/unstructured-text.py

```diff
@@ -1,26 +1,27 @@
 #
-# pip install unstructured[all-docs]
-# libmagic
+# pip install unstructured[pdf] nltk==3.8.1 huggingface_hub[hf_transfer]
 #
-# partition_object supports via unstructured library:
-#
-# "csv", "doc", "docx", "epub", "image", "md", "msg", "odt", "org",
-# "pdf", "ppt", "pptx", "rtf", "rst", "tsv", "xlsx"
+import os
+
+os.environ["HF_HUB_ENABLE_HF_TRANSFER"] = "1"
```

Comment:
[F] `pipeline` downloads `pszemraj/led-large-book-summary` so we might as well get it as quickly as possible